src/main/java/com/fcgl/madrid/forum/model/Comment.java

```diff
@@ -17,12 +17,14 @@
     private long updatedDate;
```

Comment:
Shouldn't there be a get and set for createdDate, updatedDate, likes and dislikes?

Reply:
Yes! I'll add them in

Reply:
oh sorry, didn't see it. the file viewer didn't expand everything